build.gradle

```diff
@@ -28,3 +28,7 @@ allprojects {
         mavenCentral()
     }
 }
+
+ext {
```

Comment:
any particular reason to define it here rather than `ext` in `app/build.gradle` ?

Reply:
Nothing in particular. I didn't realise we had an ext block in app gradle. It's getting used for getting properties from gradle.properties, so I think keeping the versions in a separate ext block would be good.